docs/src/understand/tedge-mapper.md

```diff
@@ -193,8 +192,7 @@ tedge mqtt sub tedge/errors
 ## Topics used by tedge-mapper
 
 - Incoming topics
-  - `tedge/measurements`
-  - `tedge/measurements/<child-id>` (for Cumulocity)
+  - `te/+/+/+/+/m/+`
 
 - Outgoing topics
   - `tedge/errors` (for errors)
```

Comment:
It will be changed soon by [#2377](https://github.com/thin-edge/thin-edge.io/issues/2377)

Reply:
We'll update the `tedge/errors` topic in another PR so we can merge this as soon as possible.

Reply:
The doc will be updated along the code: https://github.com/thin-edge/thin-edge.io/pull/2386